server/cmd/mmctl/commands/user.go

```diff
@@ -775,6 +775,12 @@ func deleteAllUsersCmdF(c client.Client, cmd *cobra.Command, args []string) erro
 	return nil
 }
 
+// userOut is the output format for users.
+type userOut struct {
+	*model.User
+	Deactivated bool
+}
```

Comment:
Using a separate struct makes the output easy to check in tests.

Reply:
0/5 but we can handle this within the template `{{if gt .DeleteAt 0 }} true {{else}} false {{end}}`.

Reply:
Sure, that would make the template easier. How would you assert the output in the tests?

Reply:
Can't you just assert the `.DeleteAt` value? The output is actually the struct itself so it should be like
```go
s.True(user.DeleteAt > 0) // Verify user is deactivated
```

Reply:
I wanted to check that the `Deactivated` field is correctly printed. Maybe the tests should use plain output instead of json one. That would make it easier to validate the output as custom types (like this one) would not be required.

Thoughts? @isacikgoz @mgdelacroix

Reply:
Yeah that also works fine, but I'm not sure if we need that level strict assertion. I'll leave it to you, my comment is more like how I would do it :)